packages/modules/src/utils/Constants.ts

```diff
@@ -21,11 +21,19 @@ export const ECDSA_OWNERSHIP_MODULE_ADDRESSES_BY_VERSION = {
   // and so on
 }
 
-export const DEFAULT_SESSION_KEY_MANAGER_MODULE = '0x000000456b395c4e107e0302553B90D1eF4a32e9'
+export const DEFAULT_SESSION_KEY_MANAGER_MODULE = '0x000002FbFfedd9B33F4E7156F2DE8D48945E7489'
 
 export const SESSION_MANAGER_MODULE_ADDRESSES_BY_VERSION = {
   V1_0_0: '0x000000456b395c4e107e0302553B90D1eF4a32e9'
-  // 'V1_0_1' : '0x000000456b395c4e107e0302553B90D1eF4a32e9'
+  // 'V1_0_1' : '0x000002FbFfedd9B33F4E7156F2DE8D48945E7489'
```

Comment:
Should uncomment this version. Default module address should also be a part of module list.

Reply:
"Default module address should also be a part of module list." didnt get this
should I not keep default address as another constant above?

Reply:
Yes we should keep the constant variable but that value should also be reflected in the versions list as well.

Reply:
you mean instead of

V1_0_0: '0x0000001c5b32F37F5beA87BDD5374eB2aC54eA8e'

doing this

export const DEFAULT_ECDSA_OWNERSHIP_MODULE = '0x0000001c5b32F37F5beA87BDD5374eB2aC54eA8e'

V1_0_0: DEFAULT_ECDSA_OWNERSHIP_MODULE

Reply:
this was already there!